Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -51,23 +54,37 @@ public class ModelGroup {
       public List<IHandModel> modelList;
       public List<IHandModel> modelsCheckedOut;
       public bool IsEnabled;
+      public bool CanDuplicate;
+
+      public ModelGroup(string groupName, List<IHandModel> modelList, HandPool handPool, bool isEnabled, bool canDuplicate) {
+        this.GroupName = groupName;
+        this.IsEnabled = isEnabled;
+        this.CanDuplicate = canDuplicate;
+        this.modelList = modelList;
+        this.modelsCheckedOut = new List<IHandModel>();
+        this._handPool = handPool;
+      }
+
       public IHandModel TryGetModel(Chirality chirality, ModelType modelType) {
         for (int i = 0; i < modelList.Count; i++) {
-          if (modelList[i].Handedness == chirality && modelList[i].HandModelType == modelType) {
+          if (modelList[i].Handedness == chirality && modelList[i].HandModelType == modelType
+            || modelList[i].Handedness == Chirality.Either && modelList[i].HandModelType == modelType) {
             IHandModel model = modelList[i];
             modelList.RemoveAt(i);
             modelsCheckedOut.Add(model);
             return model;
           }
         }
-        //Todo: if spawning enabled
-        for (int i = 0; i < modelsCheckedOut.Count; i++) {
-          if (modelsCheckedOut[i].Handedness == chirality && modelsCheckedOut[i].HandModelType == modelType) {
-            IHandModel modelToSpawn = modelsCheckedOut[i];
-            IHandModel spawnedModel = GameObject.Instantiate(modelToSpawn);
-            _handPool.modelGroupMapping.Add(spawnedModel, this);
-            modelsCheckedOut.Add(spawnedModel);
-            return spawnedModel;
+        if (CanDuplicate) {
+          for (int i = 0; i < modelsCheckedOut.Count; i++) {
+            if (modelsCheckedOut[i].Handedness == chirality && modelsCheckedOut[i].HandModelType == modelType
```

Comment:
Same check here can be simplified (might also be worthwhile to make into a helper function since it is used more than once)